sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -273,6 +287,8 @@ internal BlobContainerClient(Uri blobContainerUri, HttpPipelinePolicy authentica
             _version = options.Version;
             _clientDiagnostics = new ClientDiagnostics(options);
             _customerProvidedKey = options.CustomerProvidedKey;
+            _encryptionScope = options.EncryptionScope;
+            BlobErrors.VerifyCpkAndEncryptionScopeNotBothSet(_customerProvidedKey, _encryptionScope);
             BlobErrors.VerifyHttpsCustomerProvidedKey(_uri, _customerProvidedKey);
```

Comment:
Same minor nit - it would be nice to validate HTTPS/CPK in the same order so the same exceptions are raised.

Reply:
(Can defer)